rustfmt.toml

```diff
@@ -1,5 +1,5 @@
 # Maximum line width before rustfmt wraps code onto multiple lines
-max_width = 100
+max_width = 120
```

Comment:
can you push this back? at least for this PR? I was wondering why half of the diff in here was reformatting - it's creating a lot of noise. changes to the formatting should be a dedicated refactor pr.